docs/introduction/get-started/create-account-use-faucet.md

```diff
@@ -134,8 +139,36 @@ State synced to block 179672
       ```shell
       miden consume-notes --account <Account-Id> <Note-Id>
       ```
+5. You should see a confirmation message like this:
 
-  Amazing! You just have created a client-side zero-knowledge proof locally on your machine. 
+      ![Transaction confirmation message](../../img/get-started/transaction-confirmation.png)
+
+6. After confirming you can view the new note status by running the following command:
+
+      ```shell
+      miden notes
+      ```
+
+7. You should see something like this:
+
+      ![Viewing process info](../../img/get-started/processing-note.png)
+
+8. The note is `Processing`. This means that the note the consume transaction was sent but you haven't received confirmation of the consumption yet. You can update your view of the rollup by syncing again:
```

Comment:
I think in this sentence, there is a word missing or so.

```
This means that the note the consume transaction was sent but you haven't received confirmation of the consumption yet.
```

Is it maybe: 
```
This means that the proof of the transaction was sent, but there is no network confirmation yet
```
?